runner.yml.dist

```diff
@@ -0,0 +1,36 @@
+drupal:
+  root: "web"
+  base_url: ${env.DRUPAL_BASE_URL}
+  site:
+    profile: "oe_profile"
+    name: "Subsite"
+    generate_db_url: false
+  account:
+    name: ${env.DRUPAL_ACCOUNT_USERNAME}
+    password: ${env.DRUPAL_ACCOUNT_PASSWORD}
+
+behat:
+  tags: "~@wip"
+
+selenium:
+  host: "http://selenium"
+  port: "4444"
+  browser: "chrome"
+
+toolkit:
+  build:
+    dist:
+      commands:
+      - { task: "copy", from: "lib/modules", to: "dist/${drupal.root}/modules/custom" }
```

Comment:
Provide example with local packages instead of simlinks. Include information in README